crates/rolldown/tests/esbuild/lower/lower_nested_function_direct_eval/artifacts.snap

```diff
@@ -143,6 +143,8 @@ if (foo) {
 ## 5.js
 
 ```js
+
```

Comment:
Is this made by oxc codegen? Not a problem, but feel like it didn't manage the lines well.